server/webapp/WEB-INF/rails.new/app/presenters/api_v2/bulk_deletion_failure_result_representer.rb

```diff
@@ -0,0 +1,31 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV2
+  class BulkDeletionFailureResultRepresenter < ApiV2::BaseRepresenter
+    property :non_existent_users, exec_context: :decorator, skip_render: lambda { |array, options| array.empty? }
```

Comment:
Can you just add a quick comment in the code as to why this not-so-obvious workaround was needed?